app/controllers/api/articles_controller.rb

```diff
@@ -49,4 +49,20 @@ def show
       message: 'Article not found.'
     }, status: 404
   end
+
+  def create
+    current_admin.articles.create(
+      title: params[:title], teaser: params[:teaser], body: params[:body],
+      article_type: params[:article_type], location: params[:location], category: params[:category]
+    )
+    if current_admin.articles.last.persisted?
```

Comment:
I would have stored the article you create on line 54 in a variable and then checked if that specific variable persisted